src/chapters/setup-intro-data-types/setup.rst

```diff
@@ -1,2 +1,218 @@
 Setup For Java
 ===============
+
+For the entirety of this course, we will be coding in Java. We need to not only have
```

Comment:
Rephrasing suggestion for the second sentence: How about, "Besides installing Java on your machine, you must also add some support technologies to allow
you to run and edit Java code."